moonstone/plot/functions.py

```diff
@@ -0,0 +1,88 @@
+import sys
```

Comment:
Good that you extracted different functions in one file, it makes them more reusable. However the name `functions.py` can be somehow confusing with functions in metagenomics.

You could use a name like `utils.py` that often corresponds to utility functions. By the way, the different functions seems to be usable in part not necessary linked to `plot` module only so it could make sense to add them to the `utils` module.

I can deal with this @AgnesBaud as well as minor changes in this module like naming or comments what do you think?

Reply:
Of course ! Thank you :)